style.css

```diff
@@ -3,10 +3,10 @@
   margin: 0;
 }
 
-html,
+/* html,
 body {
```

Comment:
README has a broken image for the screenshot. Please make sure that the screenshot is an active image.